src/Microsoft.AspNet.Routing/RouteOptions.cs

```diff
@@ -15,6 +15,11 @@ public class RouteOptions
         /// </summary>
         public bool LowercaseUrls { get; set; }
 
+        /// <summary>
+        /// Gets or sets a value indicating whether a trailing slash should be appended to the generated URL.
+        /// </summary>
+        public bool AppendTrailingSlash { get; set; }
```

Comment:
`to the generated URL.` -> `to generated URLs.`

This is in the 'options' context, so we should talk about operations broadly.